src/configure-tests/feature-tests/bio_remapped.c

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/*
+ * Copyright (C) 2020 Elastio Software Inc.
+ */
+
```

Comment:
It would be great to have a comment here about kernel version from which this test is necessary, like here
https://github.com/elastio/elastio-snap/blob/de9e6bde084c1cb2fa9fde9f557a7f44388501f0/src/configure-tests/feature-tests/bio_free_pages.c#L7
or here https://github.com/elastio/elastio-snap/blob/de9e6bde084c1cb2fa9fde9f557a7f44388501f0/src/configure-tests/feature-tests/proc_create_fn_file_operations.c#L8
These comments are not present in all feature tests, but we try to include them in new tests we add.

Reply:
Done.